clang/lib/Interpreter/Wasm.cpp

```diff
@@ -109,12 +139,12 @@ llvm::Error WasmIncrementalExecutor::runCtors() const {
   return llvm::Error::success();
 }
 
-llvm::Error WasmIncrementalExecutor::cleanUp() const {
```

Comment:
We should not need this change.

Reply:
I think we would need it. I see a build error otherwise 
```
 │ │ /home/runner/work/recipes/recipes/output/bld/rattler-build_llvm_1732028457/work/clang/lib/Interpreter/Wasm.cpp:111:38: error: out-of-line definition of 'cl
 │ │ eanUp' does not match any declaration in 'clang::WasmIncrementalExecutor'
 │ │   111 | llvm::Error WasmIncrementalExecutor::cleanUp() const {
 │ │       |                                      ^~~~~~~
 │ │ /home/runner/work/recipes/recipes/output/bld/rattler-build_llvm_1732028457/work/clang/lib/Interpreter/Wasm.h:31:15: note: member declaration does not match
 │ │  because it is not const qualified
 │ │    31 |   llvm::Error cleanUp() override;
 │ │       |               ^         ~~~~~~~~
 ```
 
 CleanUp is defined like this 
 https://github.com/llvm/llvm-project/blob/b2d3cb1e7502a5ddb7d6688dce5cf51573f6189c/clang/lib/Interpreter/Wasm.h#L31